integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSessionMigration.java

```diff
@@ -293,41 +340,30 @@ private static void createAndVerifyDomain(String miiImage) {
         adminServerPodName, domainNamespace);
     checkPodExists(adminServerPodName, domainUid, domainNamespace);
 
-    // check that admin server pod is ready
+    // check that admin server pod is ready and admin service exists in the domain namespace
     logger.info("Checking that admin server pod {0} is ready in namespace {1}",
         adminServerPodName, domainNamespace);
-    checkPodReady(adminServerPodName, domainUid, domainNamespace);
-
-    // check that admin service exists in the domain namespace
-    logger.info("Checking that admin service {0} exists in namespace {1}",
-        adminServerPodName, domainNamespace);
-    checkServiceExists(adminServerPodName, domainNamespace);
+    checkPodReadyAndServiceExists(adminServerPodName, domainUid, domainNamespace);
 
     // check for managed server pods existence in the domain namespace
     for (int i = 1; i <= replicaCount; i++) {
       String managedServerPodName = managedServerPrefix + i;
 
-      // check that the managed server pod exists
-      logger.info("Checking that managed server pod {0} exists in namespace {1}",
-          managedServerPodName, domainNamespace);
-      checkPodExists(managedServerPodName, domainUid, domainNamespace);
-
-      // check that the managed server pod is ready
+      // check that the managed server pod is ready and the service exists in the domain namespace
       logger.info("Checking that managed server pod {0} is ready in namespace {1}",
           managedServerPodName, domainNamespace);
-      checkPodReady(managedServerPodName, domainUid, domainNamespace);
-
-      // check that the managed server service exists in the domain namespace
-      logger.info("Checking that managed server service {0} exists in namespace {1}",
-          managedServerPodName, domainNamespace);
-      checkServiceExists(managedServerPodName, domainNamespace);
+      checkPodReadyAndServiceExists(managedServerPodName, domainUid, domainNamespace);
     }
   }
 
   private static void createDomainCrAndVerify(String adminSecretName,
                                               String repoSecretName,
                                               String encryptionSecretName,
                                               String miiImage) {
+
+    Map<String, String> annotationKeyValues = new HashMap();
```

Comment:
Can we have tests with  multiple key.values pair with different syntax 
 Refer to https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/

Reply:
done

Reply:
done

Reply:
I want few annotation with special characters like  . (dot) _ (underscore)  and - (dash) embedded in alphanumeric string.  

See the following description on the link  

Annotations are key/value pairs. Valid annotation keys have two segments: an optional prefix and name, separated by a slash (/). The name segment is required and must be 63 characters or less, beginning and ending with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (_), dots (.), and alphanumerics between. The prefix is optional. If specified, the prefix must be a DNS subdomain: a series of DNS labels separated by dots (.), not longer than 253 characters in total, followed by a slash (/).